networkx/algorithms/tests/test_tournament.py

```diff
@@ -1,132 +1,157 @@
 """Unit tests for the :mod:`networkx.algorithms.tournament` module."""
 from itertools import combinations
-
-
+import pytest
 from networkx import DiGraph
 from networkx.algorithms.tournament import is_reachable
 from networkx.algorithms.tournament import is_strongly_connected
 from networkx.algorithms.tournament import is_tournament
 from networkx.algorithms.tournament import random_tournament
 from networkx.algorithms.tournament import hamiltonian_path
+from networkx.algorithms.tournament import score_sequence
+from networkx.algorithms.tournament import tournament_matrix
+from networkx.algorithms.tournament import index_satisfying
+
+np = pytest.importorskip("numpy")
+npt = pytest.importorskip("numpy.testing")
```

Comment:
I think these need to be moved within the individual tests where they are used, otherwise all of the tests in this module will be skipped if numpy isn't found.